packages/components/src/range-control/index.js

```diff
@@ -30,13 +30,16 @@ function RangeControl( {
 	min,
 	max,
 	setState,
+	initialValue,
 	...props
 } ) {
 	const id = `inspector-range-control-${ instanceId }`;
 	const currentInputValue = currentInput === null ? value : currentInput;
+	const initialInputValue = ! initialValue && min <= initialValue && max >= initialValue ?
+		undefined : initialValue;
 	const resetValue = () => {
 		resetCurrentInput();
-		onChange();
+		onChange( parseFloat( initialInputValue ) );
```

Comment:
I feel like there was a PR at some point to stop enforcing floats on range controls, to allow for values between 0 and 1. Not sure if that has any impact here — honestly not sure that PR existed or was merged (maybe it was just an issue?) — but I just wanted to note that in case it was implemented.

Reply:
Okay, instead I decided to parseFloat(initialValue) on line 39 to go around this.